src/style/utils/get-color-value.js

```diff
@@ -0,0 +1,8 @@
+import tokens from "@sage/design-tokens/js/base/common";
```

Comment:
Would be nice if this helper would be written in TypeScript as we're making the transition now. You don't have to rewrite everything else it's just that we have a rule that anything that is new is meant to be written in TS